apis/management.cattle.io/v3/logging_types.go

```diff
@@ -124,6 +132,9 @@ type KafkaConfig struct {
 	ZookeeperEndpoint string   `json:"zookeeperEndpoint,omitempty"`
 	BrokerEndpoints   []string `json:"brokerEndpoints,omitempty"`
 	Topic             string   `json:"topic,omitempty" norman:"required"`
+	Certificate       string   `json:"certificate"`
+	ClientCert        string   `json:"clientCert"`
+	ClientKey         string   `json:"clientKey"`
 }
```

Comment:
no SSLVerify for Zookeeper? Just wanted to make sure it's intended, as other configs have this flag set.

Reply:
Fluentd Kafka plugin does not support sslVerify yet, need to send PR to them, so not in this time